### DISCUSSION TITLE
Using or Unzipping an OVA on windows.

### DISCUSSION THREAD

**Discussion by FallowLand** [bitcraze/discussions#188] [Q&A] [answered]
The bitcraze instructions for installing VM take me through the following links:

1. https://www.bitcraze.io/documentation/tutorials/getting-started-with-crazyflie-2-x/
2. https://github.com/bitcraze/bitcraze-vm
3. https://github.com/bitcraze/bitcraze-vm/releases
This brings me to a link for an OVA file: https://files.bitcraze.io/public/vm/BitcrazeVM_2022.05.ova

My question is, how do I open an OVA file? I am using windows. Please let me know!

**Comment by krichardsson** [comment, marked as answer]
Hi!

[OVA is a standardized format for virtual machines](https://www.lifewire.com/ova-file-4144357) and can be opened by most virtualization applications, for instance [Virtual Box](https://www.virtualbox.org/) or [VMWAre Player](https://www.vmware.com/se/products/workstation-player/workstation-player-evaluation.html)

First install a virtualization application and then import the OVA in the application.

**Reply by FallowLand** [reply]
Thanks so much!
